### PR TITLE
Feature/css grid exercise

### PR DESCRIPTION
**Pull request by eithri**

### CODE REVIEW

File: 4-css-grid/grid.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Grid Exercise</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <h1>Grid Exercise</h1>
+    <div class="grid-container">
+        <nav class="block-a"></nav>
```
Comment (pushdev-code) [inline 4-css-grid/grid.html:12]:
```suggestion
        <section class="block-a"></section>
```

File: 4-css-grid/grid.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        <section class="block-b"></section>
+        <section class="block-c"></section>
+        <aside class="block-d"></aside>
```
Comment (pushdev-code) [inline 4-css-grid/grid.html:15]: Keep similar elements with the same tag.

```suggestion
        <section class="block-d"></section>
```

File: 4-css-grid/styles.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  display: grid;
+  grid-gap: 30px;
+  grid-template-columns: 250px 250px 250px;
+  grid-template-rows: 150px 150px;
+}
+
+.block-a {
+  background-image: url("https://upload.wikimedia.org/wikipedia/commons/thumb/b/b5/Yellow_triangle.svg/1200px-Yellow_triangle.svg.png");
+  background-size: cover;
+  border: 1px solid white;
```
Comment (pushdev-code) [inline 4-css-grid/styles.css:16]: you can group these properties in a new classname and reuse it, you'll avoid duplication as well.

`.block {
 background-size: cover;
  border: 1px solid white;
}`

and then in the html you do:

`<section class="block block-a"></section>`

Comment (eithri) [reply 4-css-grid/styles.css]: oh, that's right, thanks for the tip, already applied.

**Pull request merged**